aws/cf_template_test.go

```diff
@@ -614,6 +619,38 @@ func TestGenerateTemplate(t *testing.T) {
 				require.NotEqual(t, cloudformation.Integer(3), tg.UnhealthyThresholdCount)
 			},
 		},
+		{
+			name: "target port http2 when https listener and configured",
```

Comment:
@universam1 should be there a test case for GRPC as well?

Reply:
gRPC is using H2, so I guess this test is good enough, no?